.env-example

```diff
@@ -12,7 +12,10 @@ OPENAI_API_KEY=sk-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 MAX_MODEL_TOKENS=4096
 
 # Whether or not to use prefixes !gpt and !dalle
-PREFIX_ENABLED=true
+PREFIX_ENABLED=false
+
+# Automatically decide which model to use? (Don't use if prefix enabled)
```

Comment:
Can you also please add a documentation in docs/auto_model.md

Also reference it in the _sidebar.md :)

Reply:
👍